app/models/tag.rb

```diff
@@ -170,7 +170,11 @@ def self.followed_tags_for(follower:)
       .order(hotness_score: :desc)
   end
 
-  def self.followed_by(user, points = (1..))
+  # returns the tags that are followed by the user
+  # @param user [User] the user to check
+  # @param points [Range] the range of points to check for (default is 0 (including 0) to infinity)
+  # @return [ActiveRecord::Relation<Tag>] the tags followed by the user
+  def self.followed_by(user, points = (0...))
     where(
```

Comment:
@benhalpern I could use your 👀 on this please. I feel like this method should use `explicit_points` and not `points` but since I still feel a bit muddled about our points system I want to check with you on what your thoughts are?